fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -179,6 +189,51 @@ private fun TranslationsDialogHeader(
     }
 }
 
+@Composable
+private fun TranslationErrorWarning(translationError: TranslationError) {
+    val modifier = Modifier
+        .padding(top = 8.dp)
+        .fillMaxWidth()
+    val type = ReviewQualityCheckInfoType.Error
+    when (translationError) {
+        is TranslationError.CouldNotTranslateError -> {
+            ReviewQualityCheckInfoCard(
+                title = stringResource(id = R.string.translation_error_could_not_translate_warning_text),
+                type = type,
+                modifier = modifier,
+            )
+        }
+
+        is TranslationError.CouldNotLoadLanguagesError -> {
+            ReviewQualityCheckInfoCard(
+                title = stringResource(id = R.string.translation_error_could_not_load_languages_warning_text),
+                type = type,
+                modifier = modifier,
+            )
+        }
+
+        is TranslationError.LanguageNotSupportedError -> {
```

Comment:
Could you change this to the blue `info` warning? (I think this recently changed - [Figma](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?type=design&node-id=7221-189565&mode=design&t=AxN2FZCCDFR3xecF-0).)

Reply:
Done